dev/src/lobster/idents.h

```diff
@@ -1380,6 +1380,14 @@ inline string Signature(const NativeFun &nf) {
         FormatArg(r, arg.name, i, arg.type);
     }
     r += ")";
+    if (nf.retvals.size() > 0){
+        r += "->(";
+        for (auto [i, retval] : enumerate(nf.retvals)) {
+            if (i) r += ", ";
+            r += TypeName(retval.type);
+        }
+        r += ")";
```

Comment:
Since this is also used for printing certain error message, lets make it have the regular Lobster syntax that uses no `()`